Add Modivcare case study (#15262)
pulumi/docs: content/case-studies/modivcare.md

## The Challenge: When Growth Becomes a Technical Burden

By 2023, Modivcare's acquisition strategy had created a perfect storm of infrastructure complexity. Each acquired company arrived with different AWS approaches: mature Terraform and Terragrunt setups, AWS CDK or CloudFormation implementations, or no infrastructure automation at all—just manually provisioned resources and PowerShell scripts.

"We had an organization, we had a top-down design, but when there was an acquisition, if they were in AWS, that account got added to the organization after the fact," explained Zachary Cook, Senior Manager, DevOps at ModivCare.

The newly formed platform engineering team had to onboard legacy AWS accounts, invisible resources, and teams with vastly different cloud maturity levels. Traditional consolidation would require disruptive migrations that could impact healthcare services—unacceptable for a company serving vulnerable populations.

> 

Reviewer: ```suggestion

https://youtu.be/1um_8QTVqXg

```

---
title_tag: Modivcare | Case Studies
title: "Modivcare: Unified Platform in One Year"
description: |

>     Modivcare used Pulumi to transform fragmented acquisition-built infrastructure into a unified platform without service disruption, achieving 15-25% cost reductions and enabling developer self-service.

Reviewer: ```suggestion
    Modivcare used Pulumi to transform fragmented acquisition-built infrastructure into a unified platform without service disruption, achieving up to 25% cost reductions and enabling developer self-service.
```

> meta_desc: See how Modivcare used Pulumi to unify fragmented AWS infrastructure across acquisitions, reducing costs by 15-25%.

Reviewer: ```suggestion
meta_desc: See how Modivcare used Pulumi to unify fragmented AWS infrastructure across acquisitions, reducing costs by up to 25%.
```

customer_name: Modivcare
customer_logo: /images/case-studies/modivcare.png
customer_url: https://www.modivcare.com/

quote_block:
  quote: |
      "Pulumi ESC has been fundamentally game-changing in how we want to bring that apart, not just in solving that disconnected set of Terragrunt files, but also in normalizing a lot of those Terragrunt files."

>   quote_attrib: Zachary Cook, Senior Manager, DevOps at ModivCare

Reviewer: ```suggestion
  quote_attrib: Zachary Cook, Senior Manager, DevOps at Modivcare
```

>   headline_stat: 15-25% Cost Savings

Reviewer: ```suggestion
  headline_stat: 25% Cost Savings
```

>   headline: by eliminating redundant resources

Reviewer: ```suggestion
  headline: By eliminating redundant resources
```

The integration was elegantly simple: Backstage's scaffolder created repositories and pipelines, GitLab runners executed workflows, and Pulumi provisioned AWS resources. Developers gained a single interface to discover solutions and provision infrastructure without becoming infrastructure experts.

The impact on productivity was dramatic: infrastructure provisioning went from days-long bottlenecks involving multiple teams to self-service capabilities that developers could use in minutes.

## Results

Modivcare's journey delivered concrete business results:

**Operational Excellence**: Infrastructure standardization across multiple AWS accounts without service disruption. This non-disruptive approach typically reduces project delays by 70% while maintaining healthcare service reliability.

**Developer Productivity**: Self-service infrastructure eliminated development bottlenecks. Dozens of reusable modules enabled developers to focus on healthcare solutions rather than infrastructure complexity.

**Strategic Capability**: Established repeatable patterns for integrating acquisitions. Rather than creating complexity, new acquisitions now follow proven integration approaches.

> **Cost and Governance**: Visibility and standardization achieved 15-25% infrastructure cost reductions through elimination of redundant resources and improved governance.

Reviewer: ```suggestion
**Cost and Governance**: Visibility and standardization achieved up to 25% infrastructure cost reductions through elimination of redundant resources and improved governance.
```
